src/w3c/defaults.js

```diff
@@ -53,7 +53,7 @@ const w3cDefaults = {
     "privsec-section": true,
     "no-http-props": true,
   },
-  doRDFa: false,
+  doJsonLd: true,
```

Comment:
nit: false for now, till we see it in action 😉

Reply:
To be clear, we should show the benefits and ask the folks on spec-prod about enabling it by default. If people can see 👀 the benefits, I'm sure they will be excited and supportive of having this on by default.